event_lab.js

```diff
@@ -0,0 +1,27 @@
+document.addEventListener("DOMContentLoaded", () =>{
+
+    let form = document.querySelector("form")
+    form.addEventListener("submit", (e) => { //to prevent refresh refresh
```

Comment:
Having the event as an argument is part of what you need to do to prevent a refresh.

on the line below you would need to add `e.preventDefault()`

if you want to prevent the default, having it on the first line is always customary